src/main/groovy/net/vivin/gradle/versioning/VersionUtils.groovy

```diff
@@ -109,9 +109,9 @@ class VersionUtils {
         if(version.snapshot) {
             result = "$result-$version.config.snapshotSuffix"
         } else if(versionByTag.containsValue(result)) {
-            throw new BuildException("Determined version '$result' already exists in the repository at '$repository.directory'.\nFix your bumping or manually create a tag with the intended version on the commit to be released.", null)
+            throw new BuildException("Determined version '$result' already exists on a prior commit in the repository at '$repository.directory'.\nCheck your configuration to ensure that you haven't forgotten to filter out certain tags or versions. You may also be bumping the wrong component; if so, bump the component that will give you the intended version, or manually create a tag with the intended version on the commit to be released.", null)
```

Comment:
Yeah, another works - I guess what I really mean is "pre-existing"; `prior` does have a temporal semantic.